src/tests/tracing/eventcounter/runtimecounters.cs

```diff
@@ -97,10 +118,15 @@ public partial class TestRuntimeEventCounter
     {
         public static int Main(string[] args)
         {
+            AppContext.SetSwitch("appContextSwitch", true);
+            AppDomain.CurrentDomain.SetData("appContextBoolData", true); // Not loggeed, bool key
+            AppDomain.CurrentDomain.SetData("appContextBoolAsStringData", "true");
+            AppDomain.CurrentDomain.SetData("appContextStringData", "myString"); // Not logged, string does not parse as bool
```

Comment:
If possible, I'd want these in a separate test since a test failure here will get incorrectly bucketed as an `EventCounters` failure. If that's going to take too long, we can leave it here though.

Reply:
This doesn't look like it will be ready for the 12pm snap, so I can refactor.

Reply:
I'm okay leaving it as-is if you want to merge it now.

Reply:
Tests weren't complete by that point, so I think it's a moot point.

I'll move things over to a new class